src/_global_variables.scss

```diff
@@ -35,3 +49,86 @@ $active: $brand2;
 .active {
   color: $active;
 }
+
+
+// mixins
+
+@mixin transition($property:all, $duration:0.3s, $easing:ease-in-out) {
+  transition: $property $duration $easing;
+}
+
+@mixin border-radius($radius1:3px, $radius2:3px) {
+  -webkit-border-radius: $radius1 $radius2 $radius1 $radius2;
+  -moz-border-radius: $radius1 $radius2 $radius1 $radius2;
+  border-radius: $radius1 $radius2 $radius1 $radius2;
+}
+
+@mixin box-shadow($xlength:0, $ylength:2px, $size:0, $color:rgba(30,30,30,0.25)) {
+  -webkit-box-shadow: $xlength $ylength $size $color;
+  -moz-box-shadow: $xlength $ylength $size $color;
+  box-shadow: $xlength $ylength $size $color;
+}
+
+@mixin vertical-align($position: relative) {
+  position: $position;
```

Comment:
Does foundation have a class to vertically align things? I'd think so but maybe not?

Reply:
Foundation's use of Flexbox has a method to vertically center the grid cells. Docs [here](https://foundation.zurb.com/sites/docs/flex-grid.html#vertical-alignment). It's limited to:

* grid wrapper elements, e.g. block panels
* only applies to child `child` elements

The mixin is a broader little snippet (based [on this](http://zerosixthree.se/vertical-align-anything-with-just-3-lines-of-css/)) that can be used to vertically align text, icons etc, doesn't require them to be grid elements. I find it useful and worthy of inclusion, at least for reaching parity.

Reply:
makes sense. do you think we should make a class out of this e.g. `.vertical-align`?